core/modules/user/user.module

```diff
@@ -322,15 +322,12 @@ function user_load_by_name($name) {
 /**
  * Save a new or updated user.
  *
- * @param $account
- *   The user object to modify or add. If you want to modify an existing user
- *   account, you need to ensure that (a) $account is an object, and (b) you
- *   have set $account->uid to the numeric user ID of the user account you wish
- *   to modify. If you want to create a new user account, you can set
- *   $account->is_new to TRUE or omit the $account->uid field.
- *
+ * @par am $account
```

Comment:
Nice! Brief and clear, but I think it should have been `@param object $account`.

Reply:
Silly me leaving that empty space in `par am`. Thanks for noticing it. Fixed also per your recommendation I've added the reference to user-load() too.